v2/cmd/nuclei/main.go

```diff
@@ -42,7 +42,7 @@ func readConfig() {
 on extensive configurability, massive extensibility and ease of use.`)
 
 	createGroup(flagSet, "input", "Target",
-		flagSet.StringVarP(&options.Target, "target", "u", "", "target URL/host to scan"),
+		flagSet.StringSliceVarP(&options.Target, "target", "u", []string{}, "target URLs/hosts to scan"),
```

Comment:
At this point we should somehow signal for the users the difference between the cases where we accept multiple values:
1. repeating the flag -u target1 -u target2
2. comma separated values
3. cases that accept both the 1st and the 2nd approach

@ehsandeep @Ice3man543 @Mzack9999 
What do you think, should we put in parenthesis a hint on how this should be used?
e.g. "_target URLs/hosts to scan (-u target1 -u target2)_"

Reply:
> "target URLs/hosts to scan (-u target1 -u target2)"

This looks like a good indicator to me for the flags accepting values with multiple flags.

Same for the template flag as well.

> "template or template directory to use for scan (-t cves/ -t exposures/)"